### PR TITLE
CON create data directionality choice

### LINKED ISSUE #1665: Create directed CON data
I have been testing the CON WD pipeline with the data in the connectivity folder.
But the data was not directed, so I got the same results for in-degree and out-degree:

Screenshot 2023-10-19 at 08 18 12

Would be great to have directed connectivity data

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

File: braph2genesis/pipelines/connectivity directed/create_data_COND_TXT.m [hunk 1]
```diff
@@ -0,0 +1,151 @@
+function create_data_COND_TXT(data_dir, random_seed, direction)
```
Comment (annacanal) [inline braph2genesis/pipelines/connectivity directed/create_data_COND_TXT.m]: add also this file into braph2

File: braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m [hunk 2]
```diff
@@ -0,0 +1,152 @@
+function create_data_COND_XLS(data_dir, random_seed, direction)
```
Comment (annacanal) [inline braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m]: add also this file into braph2

File: braph2genesis/pipelines/connectivity directed/example_CON_WD.m [hunk 3]
```diff
@@ -5,23 +5,23 @@
 
 %% Load BrainAtlas
 im_ba = ImporterBrainAtlasXLS( ...
-    'FILE', [fileparts(which('SubjectCON')) filesep 'Example data CON XLS' filesep 'atlas.xlsx'], ...
```
Comment (annacanal) [inline braph2genesis/pipelines/connectivity directed/example_CON_WD.m:8]: update also this file in braph2

File: braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+    % saves RNG
+    rng_settings_ = rng(); rng(random_seed)
+
+    sex_options = {'Female' 'Male'};
+
+    % Group 1
+    K1 = 2; % degree (mean node degree is 2K) - group 1
+    beta1 = 0.3; % Rewiring probability - group 1
+    gr1_name = 'CON_Group_1_XLS';
```
Comment (annacanal) [inline braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m]: CON**D**_Group_1_XLS

File: braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+        % variables of interest
+        vois1 = [vois1; {sub_id, randi(90), sex_options(randi(2))}];
+    end
+    writetable(table(vois1), [data_dir filesep() gr1_name '.vois.xlsx'], 'WriteVariableNames', false)
+
+    % Group 2
+    K2 = 2; % degree (mean node degree is 2K) - group 2
+    beta2 = 0.85; % Rewiring probability - group 2
+    gr2_name = 'CON_Group_2_XLS';
```
Comment (annacanal) [inline braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m]: CON**D**_Group_2_XLS

File: braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+%  DATA_DIR folder with a specified RANDOM_SEED.
+%
+% create_data_COND_XLS(DATA_DIR, RANDOM_SEED, DIRECTION) cretes connectivity data in
+%  DATA_DIR folder with a specified RANDOM_SEED and symmetrize the matrix
+%  if DIRECTION is set to 'undirected', will preserve direction otherwise.
+%
+% See also create_data_cond_xls.
+
+if nargin < 1
+    data_dir = [fileparts(which('AnalyzeEnsemble_CON_WD')) filesep 'Example data CON XLS'];
```
Comment (annacanal) [inline braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m]: should be maybe CON**D** XLS

File: braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+end
+
+if nargin < 2
+    random_seed = 'default';
+end
+
+if nargin < 3
+    direction = 'undirected';
```
Comment (annacanal) [inline braph2genesis/pipelines/connectivity directed/create_data_COND_XLS.m]: the default should be directed, that is why this file is called COND, no?

**Review by annacanal — CHANGES_REQUESTED** [review]
By default, it should create directed data, as the name indicates (create_data_COND_XLS), and the pipeline needs. So I do not understand why the option of undirected (for that there is already the file create_data_CON_XLS)

File: braph2genesis/pipelines/connectivity directed/example_CON_WD.m (hunk 3 above)
Comment (egolol) [reply braph2genesis/pipelines/connectivity directed/example_CON_WD.m:8]: I understand that we now have to test in the exception folder, that's why it is only on genesis

**Review by annacanal — APPROVED** [review]

**Pull request merged**